lib/ios/UITabBar+centered.m

```diff
@@ -0,0 +1,53 @@
+#import "UITabBar+centered.h"
```

Comment:
These extension classes usually serve to add utility function on top of existing implementations. I personally think it's better to name these classes `...+utils.h`.
If in the future we'll need to add a few more helper functions, unrelated to centering icons we'll be able to addd them to this utils class.